packages/types/src/lesson.ts

```diff
@@ -109,8 +125,18 @@ export type CreateApiPayload = {
   duration: Duration;
 };
 
+export type UpdateApiPayload = {
+  tutorId: number;
+  lessonId: number;
+  slotId?: number;
+  start?: string;
+  duration?: Duration;
+};
+
 export type CreateLessonApiResponse = Self;
 
+export type UpdateLessonApiResponse = Self;
```

Comment:
The client is not intested in the update lesson. Omitting this will improve the performance of the server as well as the amount of the data that will be transfered between the client and the server. 

```suggestion
export type UpdateLessonApiResponse = Void;
```